Number_types/include/CGAL/cpp_float.h

```diff
@@ -169,7 +169,7 @@ class cpp_float {
     }
     // std::cout << "m = " << m << " * 2^" << exp  << std::endl;
     // fmt(m);
-    man = Mantissa(m);
+    man = std::move(m);
```

Comment:
As `m` is `uint64_t m;`, the operator for a move of `m` is a simple copy of it.

Reply:
I thought this avoids the construction of a ` Mantissa`  object that in the assignment gets copied.

Reply:
s/copied/moved/
`man = m;` should be fine.